assets/models/system/tiiuae-falcon-40b/spec.yaml

```diff
@@ -6,7 +6,7 @@ properties:
   SHA: 3d7c5902f1dc9da830979a826cd96114b3ba4ec1
   datasets: tiiuae/falcon-refinedweb
   languages: en, de, es, fr
```

Comment:
is this value correct? 24 cores, 448GB RAM, 1344GB storage --> this is what I see

Reply:
or do we mean we need 4 instances of this compute to run?

Reply:
Min SKUs added - offline conversation